### DISCUSSION TITLE
how to open a gived path file when i first open the webpage

### DISCUSSION THREAD

**Discussion by caipeng-hrv** [coder/code-server#4613] [Q&A]
<img width="1246" alt="截屏2021-12-12 下午4 54 40" src="https://user-images.githubusercontent.com/7042328/145706317-023b4dcc-20da-4200-9514-313c1fda3c16.png">

I want open the file  /Users/cp/workspace/1/EOS/测试709/netdetrans1.cpp when i open the webpage first,where the code should i change?

**Comment by lg-nineoneninethreenine** [comment]
I don't think you can open files through a query yet. @jsjoeio any ideas?

**Reply by jsjoeio** [reply]
Yeah @im-coder-lg is right :( you can open a path though i.e. `code-server <path-to-dir>`
